src/quicktype-core/language/Swift.ts

```diff
@@ -58,7 +58,9 @@ export const swiftOptions = {
         [["internal", "internal"], ["public", "public"]],
         "internal",
         "secondary"
-    )
+    ),
+    equatable: new BooleanOption("equatable", "Types conform to the Equatable protocol.", false),
+    hashable: new BooleanOption("hashable", "Types conform to the Hashable protocol.", false)
```

Comment:
I would prefer the description strings for boolean options to be phrased like:

> Make types Equatable.
> Make types Hashable.

Reply:
And no periods at the end, please.  Also, please make those two options `"secondary"` (add that as an additional argument).